nilearn/_utils/niimg_conversions.py

```diff
@@ -24,14 +24,25 @@ def _check_fov(img, affine, shape):
             np.allclose(img.get_affine(), affine))
 
 
-def _check_same_fov(img1, img2):
-    """ Return True if img1 and img2 have the same field of view
-        (shape and affine), False elsewhere.
+def _check_same_fov(*args, **kwargs):
+    """ Check the equivalence of all provided images. Parameter names are
+        used to generate user friendly error message.
```

Comment:
That function needs a better docstring. First, the notion of "equivalence" of images should be made explicit (am I right in understanding that its: same shape and affine?). Second, what the kwargs are should be detailed.

It doesn't have to be of the quality of user-facing docs, it's more a question of devs easily understanding what the function does.